pkg/api/types.go

```diff
@@ -623,8 +637,13 @@ type MultiStageTestConfigurationLiteral struct {
 	// Post is the array of test steps run after the tests finish and teardown/deprovision resources.
 	// Post steps always run, even if previous steps fail.
 	Post []LiteralTestStep `json:"post,omitempty"`
+	// Environment has the values of parameters for the steps.
+	Environment TestEnvironment `json:"env,omitempty"`
 }
 
+// TestEnvironment has the values of parameters for multi-stage tests.
+type TestEnvironment map[string]string
```

Comment:
How is this different than the current parameter interface?

Reply:
Hmm, do you mean we should use `pkg/api/parameters.go` to implement this (possibly chaining the existing `params`… parameter)? That interface is intended for dynamic values (and so is more complex), while these parameters are all static.

I don't really see an advantage in extending the parameters passed in. We cannot allow access to the extraneous parameters unless we relax the validation. The current (not present here) validation happens at configuration-resolution time, where we don't yet have information about the dynamic parameters.